<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="LoadButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>230</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Download video</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output directory: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChooseOutputButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QLabel" name="youtubetext">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Youtube URL:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>520</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Change to Downloaded videos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="URLEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Download_status">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Status: Not initiated</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
